cli/cmd/develop.go

```diff
@@ -50,8 +59,24 @@ func newForgeProjectTemplate() error {
 		}
 	}
 
+	// Map the template name to the forge template URL.
+	templateURLs := map[string]string{
+		"hello-world": "https://github.com/omni-network/hello-world-template.git",
+		"xgreeter":    "https://github.com/omni-network/omni-forge-template.git",
```

Comment:
Rather than mapping, we should just allow users to pass in the `template`. That was we can add more without upgrading cli.

If we want better names, we can rename the gitrepos that host them